react-app/src/components/CardAvatar.js

```diff
@@ -2,14 +2,14 @@ import React from 'react'
 import SignOutIcon from '../assets/sign-out.svg'
 
 function CardAvatar({ user, onSignOut }) {
-    const {name, image} = user
+    const {name, image, coins} = user
```

Comment:
Keep in mind, that `user` is potentially often encountered prop in your app, so it may make sense to move to some react context. 

Otherwise, you may find yourself dealing with *props drilling* - when you need to pass some props way down the React tree just to pass them to some low level component.